src/pybind/mgr/volumes/fs/fs_util.py

```diff
@@ -88,6 +88,28 @@ def listdir(fs, dirpath, filter_entries=None):
         raise VolumeException(-e.args[0], e.args[1])
     return dirs
 
+
+def checkdir(fs, dirpath, filter_entries=None):
```

Comment:
Can we use `listdir` from `fs/fs_util.py`?
`listdir` is already getting used to list subvolumegroups. IMO new function will add code redundancy.

Reply:
`checkdir` exits on finding the first dentry which is a directory. Hence `listdir` is not used.